Update log text (EOS to EOG)

RhinoDevel opened the pull request:
The log text "found EOS" is no longer always correct, here, because there is now an is-EOG check that also returns true for EOT.

RhinoDevel committed:
Update log text (EOS to EOG)

The log text "found EOS" is no longer always correct, here, because there is now an is-EOG check that also returns true for EOT.

Jeximo reviewed (changes requested):
"*some*" may imply many - perhaps "found **an** EOG token"

RhinoDevel committed:
Improve log msg. further by using "an" instead of "some".

As suggested, to avoid misunderstanding (no multiple EOG tokens found, just one).
